core/src/gl/shaderProgram.cpp

```diff
@@ -215,43 +223,76 @@ GLuint ShaderProgram::makeCompiledShader(const std::string& _src, GLenum _type)
 
 }
 
-void ShaderProgram::applySourceBlocks(std::string& _vertSrcOut, std::string& _fragSrcOut) {
+static const std::regex pragmaLine("^\\s*#pragma tangram:\\s+(\\w+).*$");
```

Comment:
It's still local for the compilation unit (the static is also redundant in that place). But yes, good to reduce scope where possible